changelogs/fragments/1690-ec2_ami-add-support-org_arn-orgu_arn.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
```

Comment:
I might be wrong, but these suboptions were already present in launch_permissions, am I wrong? If so, this should be more like a bugfix.

Reply:
in module [ec2_ami.py](https://github.com/ansible-collections/amazon.aws/blob/main/plugins/modules/ec2_ami.py), the options `org_arns` and `org_unit_arns` were not present before this PR

but the options `OrganizationArn` & `OrganizationalUnitArn` have been LaunchPermissions supported in boto3  from 1.20.0
https://boto3.amazonaws.com/v1/documentation/api/1.20.0/reference/services/ec2.html?highlight=modify_image_attribute#EC2.Client.modify_image_attribute

please let me know if that answers the question